Ambiguous-Squids/bangazon-api
# fix Model __str__ methods

## Description
Some Model \_\_str\_\_ methods were not returning strings

## Related Issue
N/A

## Motivation and Context
Some views were crashing with a TypeError because of the \_\_str\_\_ methods not returning strings 

## How Has This Been Tested?
All Models have had an instance created and added to DB, and then all views were loaded to check for the TypeError crash.

## Steps to Test
Make sure you have at least one instance of every Model in the database, and then load every individual models api view url (/products., /orders/, etc)

## Types of changes

## Checklist:
